extensions/interactions/CodeRepl/directives/code-repl-validation.service.spec.ts

```diff
@@ -86,8 +86,8 @@ describe('CodeReplValidationService', () => {
   });
 
   it('should catch non-string value for programming language', () => {
-    // TS ignore is used here to test warnings.
-    // @ts-ignore
+    // The customizationArguments.language.value should be a string.
```

Comment:
Not enough. You had a comment somewhere back explaining how we purposely set the value to an invalid type to check validation -- that context is useful here as well.

Reply:
done